flink-python/src/test/java/org/apache/flink/python/PythonOptionsTest.java

```diff
@@ -172,4 +172,20 @@ void testPythonClientExecutable() {
                 configuration.get(PythonOptions.PYTHON_CLIENT_EXECUTABLE);
         assertThat(actualPythonClientExecutable).isEqualTo(expectedPythonClientExecutable);
     }
+
+    @Test
+    void testPythonSystemEnvEnabled() {
```

Comment:
I don't think this test has any value. If we want to test, we should test whether the `systemEnv` obtained by the operator is different when the config is in effect.